spring-cloud-task-core/src/test/java/org/springframework/cloud/task/listener/TaskExecutionListenerTests.java

```diff
@@ -155,7 +155,7 @@ public void testAnnotationFail() {
 		context.publishEvent(new ApplicationReadyEvent(application, new String[0], context));
 
 		TaskExecution taskExecution = new TaskExecution(0, 1, "wombat", new Date(),
-				new Date(), null, new ArrayList<String>());
+				new Date(), null, new ArrayList<String>(), null);
```

Comment:
Probably want to check the errorMessage in the verifyListenerResults.  I can do this on merge.